src/components/cards/CardItem.vue

```diff
@@ -37,9 +37,25 @@
 					<ActionButton icon="icon-archive" @click="archiveUnarchiveCard()">{{ t('deck', (showArchived ? 'Unarchive card' : 'Archive card')) }}</ActionButton>
 					<ActionButton v-if="showArchived === false" icon="icon-delete" @click="deleteCard()">{{ t('deck', 'Delete card') }}</ActionButton>
 					<ActionButton icon="icon-settings-dark" @click="openCard">{{ t('deck', 'Card details') }}</ActionButton>
+					<ActionButton icon="icon-settings-dark" @click.stop="modalShow=true">{{ t('deck', 'Move card') }}</ActionButton>
 				</Actions>
 
 			</transition>
+			<modal v-if="modalShow" title="Move card to another board" @close="modalShow=false">
+				<div class="modal__content">
+					<Multiselect v-model="selectedBoard" :options="boards" label="title"
+						@select="loadStacksFromBoard" />
+					<Multiselect v-model="selectedStack" :options="stacksFromBoard" label="title" />
+					<button @click="moveCard">
+						<span class="icon-confirm" />{{ t('deck', 'Move card') }}
```

Comment:
Some design input on the buttons:
- Right-align them
- Change the order so that the Mvoe card action is the one to the most right
- Make the move card button primary
- No need for icons here I'd say